src/operator/toArray.ts

```diff
@@ -3,6 +3,24 @@ import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
 
 /**
+ * Creates an array containing all emissions of an Observable.
```

Comment:
@crunchie84 Thanks for this JSDoc.

I'd recommend that this first sentence emphasizes two key things: returns an Observable (not an array), when the source completes.

For example:

> Collects all source emissions and emits them as an array when the source completes.

Then, the informal description can be even more informal (try not mentioning closingNotifier because it assumes the reader knows the details of the buffer operator), try to imagine you're using conversational language with a co-worker:

> Emits all the values at once, as an array, when the source completes.